docs/archives/backend.md

```diff
@@ -0,0 +1,44 @@
+Several things power archives on the backend, and they're each located in different spots.
+
+These things include
+- responding to frontend fetching search results
+- responding to frontend fetching article text
+- responding to frontend fetching issue images
+- responding to frontend submitting corrections to articles
+
+## Responding to Frontend Fetching Search Results
+When the frontend makes a search request, they are directly interacting with an aws API Gateway. Read more about API Gateways below. The API Gateway then routes the frontend's request to the cloudsearch endpoint, at which point the server backing the cloudsearch endpoint will process the search query and then respond to API Gateway with the results. API Gateway then forwards this back to the frontend. You can read more about cloudsearch below.
+
+## Responding to Frontend Fetching Article Text
+There unfortunately isn't a single canonical source for article text on archives. There are two places that article text is stored. The first is on github. You can find the text for all the archived articles [here](https://github.com/thestanforddaily/archives-text). It's a massive repo and working with it is a pain (git doesn't scale super well, so actions like `git status` take > 10 min; it took me ~2 hours to submit corrections to all articles in the repo. Also, commit directly to master when working with archives-text). The other source for article text is in cloudsearch, as there is an article-text field in cloudsearch. Technically, the data in cloudsearch was derived from the archives-text repo, but they aren't connected in any other way; if you make changes to archive-text, they won't automatically sync to cloudsearch. There are many things to be improved here.
```

Comment:
Maybe we should say that the canonical source (i.e. "source of truth") for archives text is the `archives-text` github repo, but we should just always sync it to cloudsearch whenever we modify the texts in the github repo?